docs/template_plugin/src/template_plugin.cpp

```diff
@@ -52,7 +53,7 @@ Plugin::~Plugin() {
 
 std::shared_ptr<ngraph::Function> TransformNetwork(const std::shared_ptr<const ngraph::Function>& function) {
     // 1. Copy ngraph::Function first to apply some transformations which modify original ngraph::Function
-    const bool shareConsts = false, constFolding = false;
+    const bool constFolding = false;
```

Comment:
You never change this variable

Reply:
It is an example which just illustrate how you can clone your function. This variable is needed to demonstrate how to clone function with or without constant folding.

Reply:
Fixed